### DISCUSSION TITLE
Need to install memcached -- suggestions appreciated

### DISCUSSION THREAD

**Discussion by verysiberian** [oerdnj/deb.sury.org#1717] [Q&A]
Hello,

I just discovered your repository thanks to the folks at Turnkey Linux. I was able to install PHP 7.4 thanks to your repository, but I am having trouble installing memcached. I had already installed it for PHP 7.3, but I cannot seem to figure out how to install memcached for PHP 7.4 from your repo. I would appreciate any suggestions!

Best regards,
Rob

**Comment by oerdnj** [comment]
Hard to say. You'll need to describe what you are doing and what works and what doesn't in bit more detail.

**Reply by verysiberian** [reply]
Sure! I am using [TurnKey Linux's WordPress appliance](https://www.turnkeylinux.org/wordpress). It is based on Debian 10 and PHP 7.3. Your repo was recommended as a means to update to PHP 7.4, which I needed to do since my site's plugins are starting to require that version. 

I followed the instructions to use your repo and update to PHP 7.4 from [here](https://github.com/alfreddagenais/turnkeylinux-wordpress-helper/wiki/Upgrade-PHP-Wordpress-Turnkey-Linux-From-7.3-to-7.4-Debian). This worked well, and I now have PHP 7.4.

Where I am stuck is with installing a version of memcached that is PHP 7.4 compatible. I had installed memcached while still on PHP 7.3. When I try to install it now, it tells me that I already have the latest version. I am assuming that there is some way to specify that I want to install the PHP 7.4 version from your repo, but I am not sure how to do so.

Best regards,
Rob

**Reply by verysiberian** [reply]
Update: I was able to solve this by running the following commands.

apt remove php-memcached -y
apt install php7.4-memcached -y
service apache2 restart
